kubedifflib/_diff.py

```diff
@@ -77,22 +77,24 @@ def check_file(printer, path, kubeconfig=None):
   :return: Number of differences found.
   """
   with open(path, 'r') as stream:
-    expected = yaml.load(stream)
+    expected = yaml.load_all(stream)
 
-  kube_obj = KubeObject.from_dict(expected)
+    differences = 0
```

Comment:
Does all the below need indenting under the `with` block?  It seems like everything from `differences = 0` and down should be unindented by 2 spaces.